topics/gh-extension/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: GitHub
+display_name: gh extension
+github_url: https://github.com/cli/cli
+logo: gh-extension.png
+released: September 16th, 2020
+short_description: Extensions to the GitHub CLI
+topic: gh-extension
+url: https://cli.github.com
+---
+gh is GitHub's official CLI client and it can be extended by anyone through its extension system. Repositories with this topic are able to be installed, giving gh new behaviors.
```

Comment:
I can imagine that this description could be simpler; something like:

> GitHub CLI extensions are community-maintained repositories that, when installed locally, add extra functionality to gh commands.

Reply:
```suggestion
GitHub CLI extensions are community-maintained repositories that, when installed locally, add extra functionality to `gh` commands.
```